ansible/roles/ocp4-workload-open-data-hub-olm/tasks/workload.yml

```diff
@@ -0,0 +1,32 @@
+# vim: set ft=ansible
+---
+
+# Implement your Workload deployment tasks here
+- name: Create Open Data Hub Operator Subscription
+  k8s:
+    state: present
+    namespace: "{{ operator_namespace }}"
+    definition: "{{ lookup('template', 'subscription.yaml.j2' ) }}"
+
+- name: Wait for the CRD to be available
+  k8s_facts:
+    api_version: "apiextensions.k8s.io/v1beta1"
+    kind: CustomResourceDefinition
+    name: "{{ item }}"
+  loop:
+    - kfdefs.kfdef.apps.kubeflow.org
+  register: crds
+  until: crds.resources|length > 0
+  retries: 30
+  delay: 10
+
+- name: Create Namespace to deploy Open Data Hub into
+  k8s:
+    state: present
+    definition: "{{ lookup('template', 'namespace.yaml.j2') }}"
+
+- name: Deploy Open Data Hub
+  k8s:
+    state: present
+    namespace: "{{ odh_namespace }}"
+    definition: "{{ lookup('template', 'opendatahub-cr.yaml.j2') }}"
```

Comment:
It would be great to add logic to make sure it deployed successfully. Examples in the roles_ocp_workload / logging or serverless roles.